cmake-proxies/libsoxr/CMakeLists.txt

```diff
@@ -95,3 +95,6 @@ target_compile_definitions( ${TARGET} PRIVATE ${DEFINES} )
 target_compile_options( ${TARGET} PRIVATE ${OPTIONS} )
 target_include_directories( ${TARGET} PRIVATE ${INCLUDES} )
 
+if( CMAKE_CXX_COMPILER_ID STREQUAL "GNU" )
```

Comment:
I think we can have this on for CLang as well. Otherwise, I see no problems with this

Reply:
I did what was minimally necessary to fix CI builds, avoiding link time error.